### PR TITLE
Update Travis deploy

### PR DESCRIPTION
**Pull request by jessieay**
* Install plugin rather than downloading from GitHub
* Was not working bc githib repo was renamed to https://github.com/contraband/autopilot

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,8 +34,7 @@ before_deploy:
   - tar xzvf $HOME/cf.tgz -C $HOME
   - mkdir -p ${HOME}/Godeps/_workspace
```
Comment (jcscottiii) [inline .travis.yml]: remove this line

File: .travis.yml [hunk 2]
```diff
@@ -34,8 +34,7 @@ before_deploy:
...
   - export GOPATH=${HOME}/Godeps/_workspace
```
Comment (jcscottiii) [inline .travis.yml]: and remove this line. since you're not compiling from source, no need to make a `Go` workspace

Comment (jessieay) [inline .travis.yml]: thanks! updated at d0d0a77

**Comment by harrisj** [comment]
LGTM

**Pull request merged**